.github/workflows/pull_requests.yml

```diff
@@ -17,6 +20,9 @@ jobs:
           configuration-path: .github/labeler-pr-triage.yml
           repo-token: ${{ secrets.GITHUB_TOKEN }}
   NeedsTriageLabeler:
+    permissions:
+      contents: read  # for actions/labeler to determine modified files
+      pull-requests: write  # for actions/labeler to add labels to PRs
```

Comment:
The `actions/labeler` action is using a custom secret in this case, so we don't need to give it pull request write permissions.

```suggestion
```